### PR TITLE
45 knex seed database

### PR DESCRIPTION
**Pull request by Naiztu**
##### CHANGE LOG
- [x] Added seeds initial for the migrations

### CODE REVIEW

File: api/.env.example [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 PG_PORT=5432
 PG_USER=user
 PG_PASSWORD=password
 PG_DATABASE=database
 PORT=5050
-USE_AUTH=true
+USE_AUTH=false
 SECRET_KEY=secret
 JWT_EXPIRES=1d
 JWT_COOKIE_EXPIRES_IN=60
```
Comment (iangg29) [inline api/.env.example:12]: Do not change `.env.example`.

File: api/source/graphql/mutations/UserMutation.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        })
+        .catch((error: Error) => {
+          console.error(error);
+          throw new GraphQLError(error.name);
+        });
+      const newUser = await db.select().from(USER_TABLE_NAME).where({ id });
+      return newUser[0];
+    },
+  },
+};
```
Comment (iangg29) [inline api/source/graphql/mutations/UserMutation.ts:48]: User creation is no longer handled with GraphQL. (REST implementation is already created)

Delete file.

File: api/source/types/UserType.ts [hunk 3]
```diff
@@ -19,6 +19,10 @@ export const UserType = new GraphQLObjectType({
       type: GraphQLNonNull(GraphQLString),
       description: "User's name",
     },
+    role_id: {
+      type: GraphQLNonNull(GraphQLID),
+      description: "User's id rol",
+    },
     first_lastname: {
       type: GraphQLNonNull(GraphQLString),
```
Comment (iangg29) [inline api/source/types/UserType.ts:27]: Do not need to get Role ID. Role filed has already been implemented above. Remove this unnecessary field.

**Review by iangg29 — CHANGES_REQUESTED** [review]

**Comment by iangg29** [comment]
@Naiztu , tests for this PR are not passing. Please fix existing errors.

**Review by iangg29 — APPROVED** [review]
LGTM

**Review by A13xMtzL — APPROVED** [review]
Berrhy gud

**Pull request merged**